### ISSUE TITLE
Integrate External APIs and Data Services

### ISSUE THREAD

**Issue by dennisgsmith** [USACE/instrumentation#136]
- [ ] 1. API access to Survey123 and ArcGIS Online to allow manual data entry from a mobile device. (SPK)
- [ ] 2. Investigate letting users enter API parameters and instrument names to define connections easing the load on developers.
- [x] 3. CWMS (RADAR API) integration to link to data from projects using CWMS. MIDAS will show timeseries data that is collected in CWMS.

**Comment by dennisgsmith** [comment]
_Update 2023-11-09:_
- Survey123 - Waiting on activation for Survey123 on AGS04, although site now seems to load - need to check back with CWBI for verification.
- In contact with member of CWMS team regarding integration, meeting date TBD

**Comment by dennisgsmith** [comment]
Closing as completed, will open separate ticket for deployment/merge conflict resolution.
